apps/financial-aid/web-osk/src/routes/application/DataGathering/applicantDataGathering.tsx

```diff
@@ -149,58 +142,17 @@ const ApplicationInfo = () => {
         </Text>
         <Text marginBottom={3}>Lögheimili, hjúskaparstaða</Text>
 
-        <Text as="h3" variant="h5" color="blue400">
-          Skatturinn
-        </Text>
-        <Text marginBottom={3}>
-          Afrit af skattframtali og upplýsingar um staðgreiðslu í
-          staðgreiðsluskrá.
-        </Text>
+        <TaxDataGathering />
 
-        <Text marginBottom={3}>
-          Við þurfum að afla þessara gagna til að staðfesta hjá hvaða
-          sveitarfélagi þú eigir að sækja um fjárhagsaðstoð, einfalda umsóknar-
-          og vinnsluferlið og staðfesta réttleika upplýsinga.
-        </Text>
-
-        <Text marginBottom={[4, 4, 5]}>
-          Við þurfum að fá þig til að renna yfir nokkur atriði varðandi þína
-          persónuhagi og fjármál til að reikna út fjárhagsaðstoð til útgreiðslu
-          í byrjun {getNextPeriod.month}. Í lok umsóknar getur þú sent hana inn
-          eða eytt henni og öllum tengdum gögnum.
-        </Text>
+        <AboutDataGathering nextMonth={getNextPeriod.month} />
```

Comment:
why are you sending this as props when you can get straight access to this in @island.is/financial-aid/shared/lib _question_

Reply:
Nice will change, I thought for a second it had something to do with the application state but it doesn't so will move